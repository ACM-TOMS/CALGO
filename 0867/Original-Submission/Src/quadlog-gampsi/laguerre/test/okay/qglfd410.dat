### Numerical integration with qglqfd()
###
### int(sin(sigma*x) * x^alpha * exp(-x) * ln(x), x = 0..infinity)
###
### Line format: 2 sigma alpha neval result relerr abserr
###
2     -1.00000000000000000000000000000000000E+00    1.00000000000000000000000000000000000E+00         50   -3.8105372409246883772743857193304010E-02  9.41E-15 -3.59E-16
2      1.00000000000000000000000000000000000E+00    0.00000000000000000000000000000000000E+00         50   -6.9195545892028683353501448776698490E-02 -1.16E-15  8.05E-17
