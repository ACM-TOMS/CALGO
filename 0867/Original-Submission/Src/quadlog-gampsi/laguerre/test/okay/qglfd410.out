  Test integral: int(sin(sigma*x) * x^alpha * exp(-x) * ln(x), x = 0..infinity)

  1 ulp =   1.93E-34

  Floating-point precision appears to be (1 - log2(ulp) =) 113 bits

   nquad     sigma     alpha               Quadrature Result                            Exact Integral               Rel. Error RelE (ULPs)  Err (bits)
      25   -1.0000    1.0000   -3.8105372409246883772743857193304010E-02   -3.8105372409247242342435924594254717E-02   9.41E-15    4.89E+19     65.41
      25    1.0000    0.0000   -6.9195545892028683353501448776698490E-02   -6.9195545892028602849733652495807529E-02  -1.16E-15   -6.04E+18     62.39

  Maximum relative error =   9.41E-15   4.89E+19 ulps      65.41 bits at nquad =   25 sigma =   -1.0000 alpha =    1.0000
  Minimum relative error =  -1.16E-15  -6.04E+18 ulps      62.39 bits at nquad =   25 sigma =    1.0000 alpha =    0.0000
  Average relative error =   5.29E-15   2.75E+19 ulps      64.57 bits

  Done
